libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -406,20 +406,32 @@ bool WaveClip::HandleXMLTag(const std::string_view& tag, const AttributesList &a
 
 void WaveClip::HandleXMLEndTag(const std::string_view& tag)
 {
+   // All blocks were deserialized into new sequences; remove the one made
+   // by the constructor which remains empty.
+   mSequences.erase(mSequences.begin());
```

Comment:
Turned out that `WaveClip::HandleXMLEndTag` could be called twice, second call happens in `WaveTrack::HandleXMLEndTag`

Reply:
Yes!

I have pushed one more commit to fix this part.

The commit is common with #4739 where it was also necessary to address a problem Ryan found in testing.

But this is not a complete response to your review yet.